plugin-server/src/main/ingestion-queues/session-recording-v2/sessions/session-tracker.ts

```diff
@@ -0,0 +1,55 @@
+import { RedisPool } from '../../../../types'
+import { logger } from '../../../../utils/logger'
+import { SessionBatchMetrics } from './metrics'
+
+export type NewSessionCallback = (teamId: number, sessionId: string) => Promise<void>
+
+export class SessionTracker {
+    private readonly keyPrefix = '@posthog/replay/session-seen'
+    private readonly ttlSeconds = 48 * 60 * 60 // 48 hours
```

Comment:
Can we add a constant for maximum session length and make this TTL value a product of it? And if we have other places where we assume 24h, can we also refactor them to use the constant?